### PR TITLE
REVIEWS - List all reviews

### PR DESCRIPTION
**Pull request by omcmanus1**

### CODE REVIEW

File: src/Components/Reviews.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import ReviewsCard from "./ReviewsCard";
+
+export default function Reviews() {
+  const [reviewData, setReviewData] = useState([]);
+
+  useEffect(() => {
+    fetchAllReviews().then((reviews) => setReviewData(reviews));
+  }, []);
+
+  const prepReviewCard = () => {};
```
Comment (yalicee) [inline src/Components/Reviews.jsx]: rmove if not using

File: src/Components/Reviews.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import ReviewsFilter from "./ReviewsFilter";
+import { fetchAllReviews } from "../api";
+import ReviewsCard from "./ReviewsCard";
+
+export default function Reviews() {
+  const [reviewData, setReviewData] = useState([]);
+
+  useEffect(() => {
+    fetchAllReviews().then((reviews) => setReviewData(reviews));
+  }, []);
```
Comment (yalicee) [inline src/Components/Reviews.jsx]: add some loading state and functionality please

**Review by yalicee — APPROVED** [review]
Looks good but please look at comments. Also the filter functionality doesn't work for me but it's not vital to this ticket

File: src/Components/Reviews.jsx (hunk 2 above)
Comment (omcmanus1) [reply src/Components/Reviews.jsx]: done

File: src/Components/Reviews.jsx (hunk 1 above)
Comment (omcmanus1) [reply src/Components/Reviews.jsx]: worked out how to insert the function, so have amended now

**Pull request merged**